src/Storage/ClaimController.ts

```diff
@@ -53,51 +54,142 @@ export class ClaimController {
     const logger = this.logger.child({ method: 'download' })
 
     logger.trace({ ipfsHashes }, 'Downloading Claims')
-
-    await this.collection.insertMany(ipfsHashes.map(ipfsHash => ({ ipfsHash, claimId: null })), { ordered: false })
+    await this.collection.insertMany(
+      ipfsHashes.map((ipfsHash): Entry => ({
+        ipfsHash,
+        claimId: null,
+        lastDownloadAttempt: null,
+        lastDownloadSuccess: null,
```

Comment:
Why `lastDownloadSuccess`? Can there be more than one download success?

Reply:
Not really, Only in the rare case we discussed, where multiple nodes grab the same entry at the same time. So we can probably rename it so its singular.

Do we currently have any sort of naming scheme for timestamps in the database?

Here is a couple off the top of my head that I have seen used before

`lastDownloadAttemptAt` and `downloadSuccessAt`

`lastDownloadAttemptTime` and `downloadSuccessTime`

`timeOfLastDownloadAttempt` and `timeOfDownloadSuccess`